display_list/BUILD.gn

```diff
@@ -64,6 +66,7 @@ source_set("display_list") {
 
   public_deps = [
     "//flutter/fml",
+    "//flutter/impeller/runtime_stage",
```

Comment:
Keep an eye on the internal build after this lands to see if the deps needs to be added there as well.

Reply:
Wilco.